### DISCUSSION TITLE
What a different "xyz.gianlu.librespot:librespot-player:1.6.2:thin" with version without thin

### DISCUSSION THREAD

**Discussion by doquanghuyit** [librespot-org/librespot-java#548] [General]
Hi,

I'm trying to build library from source code for Android. In the sample I saw you use and version named as 1.6.2:thin
So I don't know what is different between thin and normal.

Thanks & Best Regards
Do Quang Huy

**Comment by devgianlu** [comment]
The `thin` classifier is a variant of the player jar that doesn't include dependencies and does not run standalone. The package is built here: https://github.com/librespot-org/librespot-java/blob/58defc72c2626671aeb6c436e254c046028b9a73/player/pom.xml#L81

**Reply by doquanghuyit** [reply]
Thank you very much. We already success build application from source code now.
